consensus/src/dag/types.rs

```diff
@@ -35,6 +36,23 @@ pub trait DAGMessage: Sized + Clone + Serialize + DeserializeOwned {
     }
 }
 
+#[async_trait]
+pub trait DAGNetworkSender: Send + Sync {
+    async fn send_rpc(
+        &self,
+        receiver: Author,
+        message: ConsensusMsg,
+        timeout: Duration,
+    ) -> anyhow::Result<ConsensusMsg>;
+
+    async fn send_rpc_with_retry(
```

Comment:
```suggestion
    async fn broadcast_fetch_rpc(
```

Reply:
this is not a broadcast_fetch though, it's more like exponential fallback with failures